client/signup/steps/clone-jetpack/index.jsx

```diff
@@ -0,0 +1,104 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import SignupActions from 'lib/signup/actions';
+import TileGrid from 'components/tile-grid';
+import Tile from 'components/tile-grid/tile';
+
+class CloneJetpackStep extends Component {
+	static propTypes = {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+		signupDependencies: PropTypes.object,
+	};
+
+	selectNew = () => {
+		SignupActions.submitSignupStep( { stepName: this.props.stepName }, [], {
+			cloneJetpack: 'new',
+		} );
+
+		this.props.goToNextStep();
+	};
+
+	selectMigrate = () => {
+		SignupActions.submitSignupStep( { stepName: this.props.stepName }, [], {
+			cloneJetpack: 'migrate',
+		} );
+
+		this.props.goToNextStep();
+	};
+
+	renderStepContent = () => {
+		const { originSiteName, destinationSiteName, translate } = this.props;
+
+		return (
+			<TileGrid>
+				<Tile
+					className="clone-jetpack__keep"
+					buttonLabel={ 'Keep plan where it is' }
+					description={ translate( 'Your plan would remain on %(originSiteName)s.', {
+						args: { originSiteName },
+					} ) }
+					image={ '/calypso/images/illustrations/jetpack-connection.svg' }
+					onClick={ this.selectNew }
+				/>
+				<Tile
+					className="clone-jetpack__migrate"
+					buttonLabel={ 'Migrate Jetpack plan' }
+					description={ translate(
+						'Your Jetpack plan would be migrated to the destination site, %(destinationSiteName)s.',
+						{
+							args: { destinationSiteName },
+						}
+					) }
+					image={ '/calypso/images/illustrations/jetpack-connection-migration.svg' }
+					onClick={ this.selectMigrate }
+				/>
+			</TileGrid>
+		);
+	};
+
+	render() {
+		const { flowName, stepName, positionInFlow, signupProgress, translate } = this.props;
+
+		const headerText = translate( 'Your Jetpack connection' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'Jetpack Connection' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/rewind/clone-site)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).